docs-client/src/lib/transports/annotated-http.ts

```diff
@@ -25,21 +25,22 @@ export default class AnnotatedHttpTransport extends Transport {
     return mimeType === ANNOTATED_HTTP_MIME_TYPE;
   }
 
+  public getDebugMimeType(): string {
+    return ANNOTATED_HTTP_MIME_TYPE;
+  }
+
+  protected getCurlPath(endpoint: Endpoint): string {
+    return endpoint.pathMapping.substring('exact:'.length);
```

Comment:
How about removing this method and just handling it in `DebugPage`?
I think this can cause misunderstanding to the reader because this only handles an exact path mapping.

Reply:
Removed `getCurlPath`, and I think it becomes simpler.